packages/vite/src/node/plugins/css.ts

```diff
@@ -245,7 +245,7 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         } else {
           // server only
           if (ssr) {
-            return modulesCode || `export default ${JSON.stringify(css)}`
+            return modulesCode || `export default ${JSON.stringify(null)}`
```

Comment:
**nitpick:** This would be the same as `export default null`
So why using `JSON.stringify` for this?
But even if, you may consider adding a comment to it. Maybe with a link to this PR

Reply:
Altered as suggested.

As for the comment I am reticent to add one as there is already one comment by the original author detailing the need to avoid tree shaking.